src/App.js

```diff
@@ -1,33 +1,66 @@
 import React, { Component } from 'react';
 import './App.css';
-import Button from '@material-ui/core/Button'
-import Grid from '@material-ui/core/Grid'
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
+import Login from './components/Login';
+import Main from './components/Main';
+import RideProgress from './components/RideProgress';
+import CancelConfirmation from './components/CancelConfirmation';
+import RideService from './services/ride_service';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      isLoggedIn: false
```

Comment:
nice, I always hated this as a stylistic thing

Reply:
You got it 👍